engines/bastion_katello/app/assets/javascripts/bastion_katello/products/discovery/views/yum-discovery-create.html

```diff
@@ -0,0 +1,157 @@
+<span page-title>{{ 'Create Yum Repositories' | translate }}</span>
+
+<div bst-alerts success-messages="successMessages" error-messages="errorMessages"></div>
+
+<form name="productForm" class="form" ng-disabled="creating()" novalidate role="form">
+  <section class="row">
+    <div class="col-sm-6">
+      <h4 translate>Product Options</h4>
+      <div bst-form-group>
+        <dl class="dl-horizontal dl-horizontal-left">
```

Comment:
Each of these inputs should be in its own form group and `<dl>, <dd>, and <dt>` should not be used.  See http://www.patternfly.org/pattern-library/widgets/#forms for reference.